content/dstu2/document-reference.md

```diff
@@ -13,7 +13,7 @@ title: DocumentReference | DSTU 2 API
 
 ## Create
 
-Create new documents. Only documents that would be classified as notes are currently supported.
+Create new documents. Currently only XHTML formatted documents are supported. Only documents that need to be associated to the patient's chart should be written. Please validate your document is valid XHTML using any available validator like the one linked [here]. 
```

Comment:
Probably don't need to update May. Want to deprecate and don't think anyone is using writes on May. I will update with a note rather than changing the text.